### ISSUE TITLE
MySQL Support

### ISSUE THREAD

**Issue by sbue** [chanzuckerberg/aurora-data-api#7]
Does this repo currently support mysql? I'm using this repo in conjunction with sqlalchemy-aurora-data-api and am using serverless Aurora MySQL

When I try to create tables with
`Base.metadata.create_all(bind=engine)`

This fails to create tables because it runs the statement `[SQL: DESCRIBE "users"]`, where `users` is the name of one of my tables. I'm not sure if this is known, and if so, if there's a fix.

**Comment by kislyuk** [comment]
This package does support MySQL. The test suite runs something similar to what you describe, without issues. I will take a look. Can you make a list of steps to repro this?

**Comment by ALydh** [comment]
I think i ran into the same issue, and have a small fix to make it work.

When SQLAlchemy tries to create the tables it first runs a "DESCRIBE <tables>", and if the table doesn't exist MySQL will return a error. 

This error is handled at __init__.py: 196
`return DatabaseError(MySQLErrorCodes(err_info[1]), err_info[2])`

SQLAlchemy expects the value of `err_info[1]`, so the fix was to change it to `err_info[1].value`.

**Comment by duncankoss** [comment]
I'm experiencing the same issue, and while I THINK that ALydh's solution should fix my issue, I'm having trouble testing it due to my limited ability to connect to serverless Aurora.

I've got a Lambda function that sometimes needs to dynamically create Aurora tables, and when a table doesn't already exist, my Lambda crashes. I attempted to use ALydh's fork in my Lambda function, but the github action I am using to automatically package and deploy my Lambda function fails to install direct commits from my requirements.txt file.

Are there any plans to integrate this fix and include it in the next release?
